homeassistant/components/config/config_entries.py

```diff
@@ -97,10 +97,10 @@ def get(self, request):
             flow for flow in hass.config_entries.flow.async_progress()
             if flow['source'] != config_entries.SOURCE_USER])
 
-    @asyncio.coroutine
     @RequestDataValidator(vol.Schema({
         vol.Required('domain'): str,
     }))
+    @asyncio.coroutine
```

Comment:
Since this is covered by tests, we could also just add `async` keyword to function.